vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/facade/ZephyrFacade.java

```diff
@@ -144,15 +148,15 @@ private String findFolderId(String cycleId, String projectAndVersionUrlQuery)
         return folderId.get(0).toString();
     }
 
-    private Map<TestCaseStatus, Integer> getExecutionStatuses() throws IOException, JiraConfigurationException
+    private Map<TestCaseStatus, String> getExecutionStatuses() throws IOException, JiraConfigurationException
     {
         String json = getJiraClient().executeGet(ZAPI_ENDPOINT + "util/testExecutionStatus");
-        Map<TestCaseStatus, Integer> testStatusPerZephyrIdMapping = new EnumMap<>(TestCaseStatus.class);
+        Map<TestCaseStatus, String> testStatusPerZephyrIdMapping = new EnumMap<>(TestCaseStatus.class);
         zephyrExporterConfiguration.getStatuses().entrySet().forEach(s ->
         {
             List<Integer> statusId = JsonPathUtils.getData(json, String.format("$.[?(@.name=='%s')].id", s.getValue()));
             notEmpty(statusId, "Status '%s' does not exist", s.getValue());
-            testStatusPerZephyrIdMapping.put(s.getKey(), statusId.get(0));
+            testStatusPerZephyrIdMapping.put(s.getKey(), String.valueOf(statusId.get(0)));
```

Comment:
Can you please elaborate what is the reason for this conversion?

Reply:
https://github.com/vividus-framework/vividus/pull/2725#discussion_r896928395

Reply:
This change requires due to `Scale` return and use string value of test cases statuses, when `Squad` operate with id values of integer type.